skribenten-backend/src/main/kotlin/no/nav/pensjon/brev/skribenten/services/BrevredigeringService.kt

```diff
@@ -271,6 +329,39 @@ class BrevredigeringService(
         } else null
     }
 
+    private suspend fun validerVedtaksbrevAttestert(brev: Dto.Brevredigering, brevtype: LetterMetadata.Brevtype) {
+        if (Features.attestant.isEnabled() && brevtype == LetterMetadata.Brevtype.VEDTAKSBREV && brev.info.attestertAv == null) {
+            throw BrevIkkeKlartTilSendingException("Brevet ${brev.info.id} er ikke attestert.")
+        }
+    }
+
+    private suspend fun medAttestertVedtaksbrev(saksId: Long, brevId: Long, attester: suspend (UserPrincipal, Dto.Brevredigering) -> ServiceResult<Dto.Brevredigering>?) {
```

Comment:
Det er noe som ikke klinger helt her. Navnet på funksjonen får meg til å tenke at database oppdateringen burde skje her. Siden brevet vil ikke faktisk være attestert om det ikke skjer. Jeg tenker kanskje at etter at du innførte `validerVedtaksbrevAttestert` så er ikke denne funksjonen nødvendig, og burde bare inlines i den public `attester` funksjonen.

Reply:
For så vidt einig, men synes attester-metoden blir for lang og med for mange separate ansvarsområde. Men vi kan jo starte der og heller iterere seinare.